### PR TITLE
Added WindowsUserJumpLists artifact definitions

### PR DESCRIPTION
**Pull request by ant1**

### CODE REVIEW

File: data/windows.yaml [hunk 1]
```diff
@@ -2768,6 +2768,20 @@ sources:
 labels: [Users]
 supported_os: [Windows]
 ---
+name: WindowsUserJumpLists
```
Comment (joachimmetz) [inline data/windows.yaml]: Please make this a group and split the different jumplist types into separate definitions 

This allows additions of other (lesser known) jump list formats, such as Explorer ProgramsCache Registry values. Also see: https://github.com/libyal/dtformats/blob/master/documentation/Jump%20lists%20format.asciidoc

**Comment by joachimmetz** [comment]
@ant1 thx for the addition, though I have a request to change the PR a bit.

File: data/windows.yaml (hunk 1 above)
Comment (ant1) [reply data/windows.yaml]: This should be fixed

**Comment by joachimmetz** [comment]
@ant1 excellent, thx. I'll have a closer look later today.

**Pull request merged**